be/test/storage/lake/tablet_manager_test.cpp

```diff
@@ -61,7 +61,7 @@ class LakeTabletManagerTest : public testing::Test {
         CHECK_OK(FileSystem::Default()->create_dir_recursive(_location_provider->segment_root_location(1)));
         _mem_tracker = std::make_unique<MemTracker>(1024 * 1024);
         _update_manager = std::make_unique<lake::UpdateManager>(_location_provider, _mem_tracker.get());
-        _tablet_manager = new starrocks::lake::TabletManager(_location_provider, _update_manager.get(), 16384);
+        _tablet_manager = new starrocks::lake::TabletManager(_location_provider, _update_manager.get(), 1024 * 1024);
```

Comment:
the main change.
https://github.com/StarRocks/starrocks/pull/56209 pr changed `charge` calculation algorithm, resulting in a little bigger. 
if the capacity is too small, the cache item will be cleaned up immediately after insert.